pkgs/by-name/sy/sysmenu/package.nix

```
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  git,
+  gtkmm4,
+  gtk4-layer-shell,
+  wrapGAppsHook,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation {
+  pname = "sysmenu";
+  version = "unstable-13-06-2024";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "sysmenu";
+    rev = "0b891e5d27f286b867d4a0984ee284a3456de851";
+    sha256 = "sha256-zI6dmS+ZqcGrG/joWouqrMNQfTRflP677kYJLJEWTBc=";
+  };
+
+  patches = [
+    ./001-no-runtime-config.patch
+    ./002-cflags-fix.patch
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    git
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtkmm4
+    gtk4-layer-shell
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make sysmenu
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 sysmenu $out/bin/sysmenu
+    runHook postInstall
+  '';
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "Simple program launcher written in gtkmm 4";
```

I think the only thing that matters here is that this program uses GTK4
```suggestion
    description = "Simple program launcher using GTK4";
```